Dockerfile.mysql

```diff
@@ -0,0 +1,7 @@
+FROM mysql:latest
+
+ENV MYSQL_DATABASE=app_db
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
```

Comment:
The `VOLUME` instruction is missing the correct syntax. It should be `VOLUME ["/var/lib/mysql"]` to properly specify the volume path.